dipy/workflows/stats.py

```diff
@@ -527,9 +528,13 @@ def run(self, subject_folder, threshold=6, out_dir=''):
             Path to the input subject folder. This path may contain
             wildcards to process multiple inputs at once.
 
+        clust_thr : string (default [5,3,1.5]), optional
```

Comment:
`variable float` instead of `string`

Reply:
should I call workflow like this:
`dipy_buan_shapes subjects_small/ --clust_thr [5,3,1.25] --out_dir "sm_test"`

or

`dipy_buan_shapes subjects_small/ --clust_thr "[5,3,1.25]" --out_dir "sm_test"`

both are giving me an error

Reply:
got it, like this:

`dipy_buan_shapes subjects_small/ --clust_thr 5 3 1.25 --out_dir "sm_test"`

Thank you @skoudoro